### PR TITLE
Information update

### PR DESCRIPTION
**Pull request by joinimran**
Deleting old information and redirecting the user to the current and updated document.

Problem: https://github.com/MicrosoftDocs/windows-itpro-docs/issues/5514

### CODE REVIEW

File: windows/security/threat-protection/microsoft-defender-atp/microsoft-cloud-app-security-config.md [hunk 1]
```diff
@@ -39,21 +43,7 @@ Once activated, Microsoft Defender ATP will immediately start forwarding discove
...
-3. Select **Win10 Endpoint Users report**, which contains the data coming from Microsoft Defender ATP.
-
-    ![Win10 endpoint users](./images/win10-endpoint-users.png)
-
-This report is similar to the existing discovery report with one major difference: you can now benefit from visibility to the machine context.
-
-Notice the new **Machines** tab that allows you to view the data split to the device dimensions. This is available in the main report page or any subpage (for example, when drilling down to a specific cloud app).
-
-![Cloud discovery](./images/cloud-discovery.png)
+To view and access Microsoft Defender ATP data in Microsoft Cloud Apps Security see [Investigate machines in Cloud App Security](https://docs.microsoft.com/en-us/cloud-app-security/wdatp-integration#investigate-machines-in-cloud-app-security).
```
Comment (illfated) [inline windows/security/threat-protection/microsoft-defender-atp/microsoft-cloud-app-security-config.md]:
```suggestion
To view and access Microsoft Defender ATP data in Microsoft Cloud Apps Security, see [Investigate machines in Cloud App Security](https://docs.microsoft.com/en-us/cloud-app-security/wdatp-integration#investigate-machines-in-cloud-app-security).
```
minor: comma added

**Comment by jvsam** [comment]
@JohanFreelancer9 for copy review. Thank you.

File: windows/security/threat-protection/microsoft-defender-atp/microsoft-cloud-app-security-config.md [hunk 2]
```diff
@@ -31,6 +31,10 @@ To benefit from Microsoft Defender Advanced Threat Protection (ATP) cloud app di
 >[!NOTE]
 >This feature will be available with an E5 license for [Enterprise Mobility + Security](https://www.microsoft.com/cloud-platform/enterprise-mobility-security) on machines running Windows 10, version 1709 (OS Build 16299.1085 with [KB4493441](https://support.microsoft.com/help/4493441)), Windows 10, version 1803 (OS Build 17134.704 with [KB4493464](https://support.microsoft.com/help/4493464)), Windows 10, version 1809 (OS Build 17763.379 with [KB4489899](https://support.microsoft.com/help/4489899)) or later Windows 10 versions.
 
+> See [Microsoft Defender Advanced Threat Protection integration with Microsoft Cloud App Security](https://docs.microsoft.com/en-us/cloud-app-security/wdatp-integration) for detailed integration of Microsoft Defender ATP with Microsoft Cloud App Security.
```
Comment (JohanFreelancer9) [inline windows/security/threat-protection/microsoft-defender-atp/microsoft-cloud-app-security-config.md]:
```suggestion
> See [Microsoft Defender Advanced Threat Protection integration with Microsoft Cloud App Security](https://docs.microsoft.com/cloud-app-security/wdatp-integration) for detailed integration of Microsoft Defender ATP with Microsoft Cloud App Security. 
```

File: windows/security/threat-protection/microsoft-defender-atp/microsoft-cloud-app-security-config.md (hunk 1 above)
Comment (JohanFreelancer9) [inline windows/security/threat-protection/microsoft-defender-atp/microsoft-cloud-app-security-config.md]:
```suggestion
To view and access Microsoft Defender ATP data in Microsoft Cloud Apps Security, see [Investigate machines in Cloud App Security](https://docs.microsoft.com/cloud-app-security/wdatp-integration#investigate-machines-in-cloud-app-security).
```

**Comment by joinimran** [comment]
Ready to go live.

**Comment by illfated** [comment]
#sign-off

**Pull request merged**